assets/prometheus-k8s/prometheus-k8s-service-monitor-availability-app-create.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: monitor-app-create
+  labels:
+    k8s-app: monitor-app-create
+spec:
+  jobLabel: k8s-app
+  endpoints:
+  - port: http-metrics
```

Comment:
`http-metrics` -> `https-metrics`

Reply:
@brancz 

> http-metrics -> https-metrics

This is actually [referentially correct](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_monitor_availability/files/monitor-app-create.yaml#L79) but I agree I goofed with the name.

>I'm struggling to understand the purpose of this without alerting rules or other interactions with the metrics. Do we just not have an idea yet what we would want to alert on or is it purely for reporting purposes?

Primary interaction with the metrics at this moment would be manual analysis. We're still iterating on the metrics themselves and the subsequent alerts.

I suggest it would be helpful clearly articulate the acceptance criteria for ServiceMonitor and rule additions to the operator.

Reply:
My goal was to get the bare minimum integration set up to facilitate experimentation and iteration, but I'm sure there's an argument to be made for WIP'ing the PR until such experimentation is done, which is fine. What's your take on it? In any case I would like to make it clearly stated for the benefit of others.

Reply:
> Primary interaction with the metrics at this moment would be manual analysis.

This is a perfectly reasonable argument to have, just wanted to make sure we're not creating something that doesn't end up not having targets or something like that.